custom/ewsghana/handlers/alerts.py

```diff
@@ -293,8 +293,11 @@ def handle(self):
             text = self.msg.text
 
         if not domain.commtrack_enabled:
-            print "tutaj?"
             return False
+
+        if not verified_contact.location_id:
```

Comment:
would it be better to check self.location here since that's what is used later on? also, what's the right behavior if the location is archived, should we send the same message?